src/main/resources/META-INF/third-party-licenses/REVISED_BSD.txt

```diff
@@ -1,30 +0,0 @@
-JSch 0.0.* was released under the GNU LGPL license.  Later, we have switched
```

Comment:
is it Arrow related? how to know?

Reply:
This PR reviews other dependencies, not just Arrow. This license file is removed because the jsch (SSH client pulled in by hadoop-common) is being removed.